packages/blade/src/components/Typography/Heading/Heading.tsx

```diff
@@ -83,7 +83,7 @@ const getProps = <T extends { variant: HeadingVariant }>({
   return props;
 };
 
-const Heading = <T extends { variant: HeadingVariant }>({
+export const Heading = <T extends { variant: HeadingVariant }>({
```

Comment:
So we're moving to named exports everywhere now?

Reply:
the problem is right now we end up in a lot of mess sometimes because of named export and the weird `.web.tsx` and `.native.tsx` and then to resolve them there's no option without having named exports, i guess both me and Anurag have struggled because of this. so trying if we can use named everywhere then. because in the end for consumers everything is named import only. it's just for internals, open for the conclusion we come to mutually

Reply:
We can probably enforce this inside `components/*` with a linting rule

Reply:
Enforce named exports?